### PR TITLE
Add Address selector to placeholder

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Placeholders now have selector elements to find fields in Addresses  

N.B I do have reservations over the proposed solution names as explained on the validator PR

### How to review 
Using test_address make sure on entering a full address on the first page that only the first line appears on the confirmation page

### CODE REVIEW

File: scripts/run_validator.sh
Comment (MebinAbraham) [inline scripts/run_validator.sh:2]: Reminder to revert before merge.

File: test_schemas/en/test_address.json
Comment (MebinAbraham) [inline test_schemas/en/test_address.json:68]: Could be a confirmation question

**Review by MebinAbraham — COMMENTED** [review]
Feature works as expected, will approve once validator has been addressed.

File: app/questionnaire/placeholder_parser.py
Comment (ajmaddaford) [inline app/questionnaire/placeholder_parser.py:88]: It's not been introduced by this change, but this highlights that the values in `dict` answers aren't being escaped. We can either address it in this pull request, or raise a card for it?

Comment (ajmaddaford) [reply app/questionnaire/placeholder_parser.py:88]: You can test this by adding html into the address line 1 answer and seeing it rendered on the confirmation page.

File: test_schemas/en/test_address.json
Comment (LJBabbage) [reply test_schemas/en/test_address.json:68]: I reckon it will be ok, considering its a test schema

File: app/questionnaire/placeholder_parser.py
Comment (LJBabbage) [reply app/questionnaire/placeholder_parser.py:88]: If we would usually store HTML and escaped on playback, then it realistically should be this card. The only reason I therefore would suggest a different story would be because of the testing deadline for DAC tomorrow. It wouldn't matter to them

**Review by MebinAbraham — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**
